ee/app/livechat-enterprise/server/settings.js

```diff
@@ -90,6 +90,18 @@ export const createSettings = () => {
 		],
 	});
 
+	settings.add('Livechat_auto_transfer_chat_timeout', 0, {
```

Comment:
Let's change the setting value from 'minutes' to 'seconds', okay?

Reply:
Yeah makes sense, it gives more control.